test/fixtures/get-orders-response.json

```diff
@@ -0,0 +1,307 @@
+[
+  {
+    "specification": {
+      "direction": "buy",
+      "quantity": {
+        "currency": "USD",
+        "value": "1122.990930900328",
+        "counterparty": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q"
+      },
+      "totalPrice": {
+        "currency": "EUR",
+        "value": "17.70155237781915",
+        "counterparty": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q"
+      },
+      "immediateOrCancel": true
+    },
+    "state": {
+      "sequence": 719930
+    }
+  },
+  {
+    "specification": {
+      "direction": "buy",
+      "quantity": {
+        "currency": "EUR",
+        "value": "750",
+        "counterparty": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q"
+      },
+      "totalPrice": {
+        "currency": "USD",
+        "value": "19.11697137482289",
+        "counterparty": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q"
+      }
+    },
+    "state": {
+      "sequence": 756999
+    }
+  },
+  {
+    "specification": {
+      "direction": "buy",
+      "quantity": {
+        "currency": "USD",
+        "value": "19.50899530491766",
+        "counterparty": "rpDMez6pm6dBve2TJsmDpv7Yae6V5Pyvy2"
+      },
+      "totalPrice": {
+        "currency": "USD",
+        "value": "18.46856867857617",
+        "counterparty": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q"
+      }
+    },
+    "state": {
+      "sequence": 757002
+    }
+  },
+  {
+    "specification": {
```

Comment:
Why the specification and state split? It seems like specification includes fields that can change, as an offer gets partially executed rather than "this is the order as placed/specified".